script/build

```diff
@@ -16,3 +16,6 @@ npx copyfiles -u 1 "./lib/**/*.d.ts" ./lib-esm
 
 # Copy css
 npx copyfiles --flat ./lib/components.css ./css
+
+# Build components.json file for documentation
+npx ts-node script/components-json/build
```

Comment:
I am just not sure if I understand the reasoning behind of including the command here? I would appreciate if you could clarify, thank you 🙏🏻

Reply:
@broccolinisoup this command should mirror what is being run currently in the CI job that commits back to the Pull Request with a generated `components.json` file (I hope 🤞 lol). With this change, we are no longer committing this file to the repo and so need to generate it before the package is published so that it's available on npm. Hope that makes sense 👀